### ISSUE TITLE
websetroot: Completely fails fresh install of Twisted 2.0.0 and Twisted Web 0.5.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11502]
|<img alt="levi's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| levi reported|
|-|-|
|Trac ID|trac#945|
|Type|defect|
|Created|2005-03-24 18:01:37Z|

websetroot is useless and buggy and needs to be removed.

```
radix@takkun ~% ./Projects/Twisted/trunk/bin/web/websetroot 
Traceback (most recent call last):
  File "./Projects/Twisted/trunk/bin/web/websetroot", line 21, in ?
    run()
  File "/Users/radix/Projects/Twisted/trunk/twisted/web/scripts/websetroot.py", line 78, in run
    main(config)
  File "/Users/radix/Projects/Twisted/trunk/twisted/web/scripts/websetroot.py", line 67, in main
    application = twistd.loadApplication(config, passphrase)
AttributeError: 'module' object has no attribute 'loadApplication'
```

<details><summary>Searchable metadata</summary>

```
trac-id__945 945
type__defect defect
reporter__levi levi
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1111687297000000 1111687297000000
changetime__1149391034000000 1149391034000000
version__ 
owner__ 
cc__radix cc__exarkun cc__jknight cc__levi cc__jerub
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="levi's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|levi commented|
|-|-|

```
#!html
<pre>
levi@piglet:~$ websetroot
Traceback (most recent call last):
  File "/usr/bin/websetroot", line 21, in ?
    run()
  File "/usr/lib/python2.4/site-packages/twisted/web/scripts/websetroot.py",
line 78, in run
    main(config)
  File "/usr/lib/python2.4/site-packages/twisted/web/scripts/websetroot.py",
line 67, in main
    application = twistd.loadApplication(config, passphrase)
AttributeError: 'module' object has no attribute 'loadApplication'
==========================

I would offer a patch, but I'm not entirely sure I know what websetroot is
trying to be. (Since it's on my system path, a man page might be nice :)
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="levi's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|levi commented|
|-|-|

```
#!html
<pre>
C:\websetroot
Traceback (most recent call last):
  File "C:\Python24\scripts\websetroot.py", line 20, in ?
    from twisted.web.scripts.websetroot import run
  File "C:\Python24\Lib\site-packages\twisted\web\scripts\websetroot.py", line
5, in ?
    from twisted.scripts import twistd
  File "C:\Python24\Lib\site-packages\twisted\scripts\twistd.py", line 6, in ?
    from twisted.python import log, syslog
  File "C:\Python24\Lib\site-packages\twisted\python\syslog.py", line 5, in ?
    syslog = __import__('syslog')
ImportError: No module named syslog
==========================

I'm still guessing on what websetroot is, so I'm not sure if it's supposed to
work on windows, but it's not happy either. FYI.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
It's a little utility for changing the root that a web.tap is serving out 
of. It's a silly program and should probably be removed.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set owner to @jerub|
|-|-|

Up for review in remeove-websetroot-945 [sic]

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

trac is so hard to use

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @radix|
|-|-|

Looks good.  Merge away.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set owner to @exarkun|
|-|-|

It wasn't good

I had to delete doc/web/man because it was breaking process-docs


Plz rereview

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @radix|
|-|-|

Looks better now.  Python 2.3 slave actually succeeds on the process-docs step now.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set status to `closed`|
|-|-|

(In [17047]) Merge remeove-websetroot-945; Remove websetroot and all remaining traces - take 2

Author: radix
Reviewer: exarkun
Fixes [#11502](https://github.com/twisted/twisted/issues/11502)

websetroot was broken for a long time and had dubious utility anyway.
delete it, and this time make sure we don't break process-docs

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
